onnxruntime/contrib_ops/cpu/crop_and_resize.cc

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The copyright notice should say that modifications are from Microsoft. Please check roialign.cc for an example.

Reply:
revised following roialign.cc